app.config.ovm.json

```diff
@@ -0,0 +1,16 @@
+{
+  "contractsDir": "contracts",
+  "artifactsDir": "./build/contracts",
+  "contractAbstractionDir": "./build",
+  "useDockerisedSolc": false,
+  "compilerSettings": {
```

Comment:
Are any of these contracts likely to be compiled and used by the core code? If so, we'll need to make sure that the OVM compiler doesn't append metadata to the bytecode it compiles.

Recently we removed metadata from being appended to the bytecode, and removed the handling of it from the gethwrapper generation scripts. I'm not sure if the ovm compiler has the same option, but we will need it if the node uses any of these abis to generate wrappers:

```json
    "metadata": {
      "bytecodeHash": "none"
    }
```

Reply:
As far as I know, core only uses LinkToken v0.4 to generate wrappers from this repo. It also uses LinkToken v0.6 (now migrated to v0.7 here) for core OVM tests, but not for wrapper generation.

Reply:
@alexroan I've removed the solc metadata generation [here](https://github.com/smartcontractkit/LinkToken/pull/28/commits/04735e7c77fb3ca572e21140af86bcc6686e965e).